addons/website/controllers/main.py

```diff
@@ -291,6 +292,45 @@ def get_dynamic_snippet_templates(self, filter_id=False):
         )
         return templates
 
+    @http.route('/website/get_meta_from_url', type='json', auth='user', website=True)
+    def get_meta_from_url(self, url):
```

Comment:
on avait pas dis client side ?

Reply:
and just favicon / title

Reply:
Cette route n'est pas utilisée c'est du old code de l'implémentation précédente, je le garde sous la main tant que la PR n'est pas ready, comme le commit 'wip link in page' qui n'est plus pertinent.